google/auth/_helpers.py

```diff
@@ -0,0 +1,65 @@
+# Copyright 2015 Google Inc.
```

Comment:
Since this is based on code from oauth2client that's (c) 2015, it's important not to modify the date.